app/src/org/commcare/mediadownload/MissingMediaDownloadHelper.kt

```diff
@@ -171,7 +177,7 @@ object MissingMediaDownloadHelper : TableStateListener {
     }
 
 
-    private suspend fun downloadMissingMediaResource(uri: String): MissingMediaDownloadResult {
+    private fun downloadMissingMediaResource(uri: String): MissingMediaDownloadResult {
```

Comment:
Also wondering why we removed `suspend` here?

Reply:
`suspend` is only required when we launch a corountine from this method or this method calls another `suspend` functions. So it didn't serve any purpose to declare this method as `suspend`